### ISSUE TITLE
Wrong edit link in docs and strange character

### ISSUE THREAD

**Issue by adrianlzt** [logstash-plugins/logstash-filter-dissect#43]
In the [official doc](https://www.elastic.co/guide/en/logstash/master/plugins-filters-dissect.html) the edit link points to the dns plugin: https://github.com/logstash-plugins/logstash-filter-dns/edit/master/docs/index.asciidoc

Also, in the description there are two strange `#`:
```
The Dissect filter is ... set of delimiters # to a string value... field values with # more regex...
```

**Comment by guyboertje** [comment]
@adrianlzt 
Thanks for pointing this out.
I will create a docs PR to fix the second (`#`) bits.

And weirdly the filter-dns page edit link points at filter-dissect!
The edit links are provided by some include voodoo that I have not seen yet. I will speak to @dedemorton.

**Comment by dedemorton** [comment]
@guyboertje Unfortunately I fixed this once and submitted a PR, but things moved underneath me and the change never got merged. I will submit another fix to the edit URLs this week.

**Comment by guyboertje** [comment]
closed by #44
